open-apparel-registry/open-apparel-registry
# API documentation updates

## Overview

Fix an endpoint summary and add a tool to generate Markdown from OpenAPI 2 JSON

Connects #1797

## Demo


<img width="324" alt="Screen Shot 2022-05-06 at 4 59 34 PM" src="https://user-images.githubusercontent.com/17363/167229319-5afb7594-4f74-4859-8de9-7864a45f3fba.png">

## Testing Instructions

* Download http://localhost:808/api/docs/?format=openapi to `oar.json`
* Run `./tools/openapi2md oar.json` > oar.md
* Verify that `oar.md` is a valid markdown document
* Review the updated [API Document ion Google Doc](https://docs.google.com/document/d/1qJBTJCRv-kvmkGtRKpMT_iENa9oBrdaplmwrskLb3ds/edit#heading=h.s7q0cau43tnh)

## Checklist
